rest_framework/renderers.py

```diff
@@ -421,6 +421,14 @@ def show_form_for_method(self, view, method, request, obj):
             return False  # Doesn't have permissions
         return True
 
+    def _get_serializer(self, serializer_class, view_instance, request, *args, **kwargs):
+        kwargs['context'] = {
```

Comment:
In theory, can't we get this [from `view.get_serializer_context`](https://github.com/tomchristie/django-rest-framework/blob/c9618cc0f33bec0e14c814e778727c084410badb/rest_framework/generics.py#L129-L137)?